src/aind_data_access_api/document_db.py

```diff
@@ -494,7 +526,7 @@ def _record_to_operation(record: str, record_id: str) -> dict:
     def upsert_list_of_docdb_records(
         self,
         records: List[dict],
-        max_payload_size: int = 2e6,
+        max_payload_size: int = 5e6,
```

Comment:
updated to 5MB after testing. This can write approx 400 records per request. We can optionally look into compressing the request payload in the future.